web-apis/bg/spell-checker.js

```diff
@@ -0,0 +1,29 @@
+const spellchecker = require('spellchecker')
+const spellCheckerLib = require('../../lib/spell-checker')
+
+let self = module.exports = {
+  spellCheck(text) {
+    return !self.isMisspelled(text)
+  },
+  isMisspelled(text) {
+    const misspelled = spellchecker.isMisspelled(text)
+
+    // Makes everything faster.
+    if (!misspelled) {
+      return false
+    }
+
+    // Check the locale and skip list.
+    if (locale.match(/^en/) && SKIP_LIST.includes(text)) {
```

Comment:
SKIP_LIST should be exported by `/lib/spell-checker.js` rather than put on the global, so that this line will become `spellCheckerLib.SKIP_LIST`